src/components/Promotions/PromotionCard/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {PromotionCardProps} from "./types";
+import {PromotionCardStyles} from "./styles";
+import CustomLink from '../../common/CustomLink';
+
+const PromotionCard = ({content}: PromotionCardProps) => {
+    const {image, title, link} = content
+
+    return (
+        <PromotionCardStyles>
+            <div className="card-image">
+                <img src={image.url} alt={image.alt}/>
```

Comment:
better to use ? syntax or add condition to verify that image is object and it has .url property